.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
I looked into the codes and found out those localPrefixes in the rules cannot be remove, otherwise tests will all fail. @notmessenger